### PR TITLE
[api] Update last_login in ObtainAuthTokenView #112

### LINKED ISSUE #112: [radius/api] Update last login date in ObtainAuthTokenView
Update the last login date in ObtainAuthTokenView after successful authentication.

### PR DESCRIPTION
**Pull request by atb00ker**
Closes #112

### CODE REVIEW

File: openwisp_radius/api/views.py [hunk 1]
```diff
@@ -468,6 +469,8 @@ def post(self, request, *args, **kwargs):
         user = self.get_user(serializer, *args, **kwargs)
         token, created = UserToken.objects.get_or_create(user=user)
         radius_token = self.get_or_create_radius_token(user)
+        user.last_login = timezone.now()
+        user.save(update_fields=['last_login'])
```
Comment (nemesisdesign) [inline openwisp_radius/api/views.py:473]: I tried commenting out these two lines and the test passes anyway, can you investigate why?
I thought this wasn't being done by the token serializer but I may have been wrong

Comment (atb00ker) [reply openwisp_radius/api/views.py:473]: Amazing, lol, I'll check it out! :smile:

Comment (atb00ker) [reply openwisp_radius/api/views.py:473]: The testcase had a problem; fixed it.

File: openwisp_radius/api/views.py [hunk 2]
```diff
@@ -502,6 +505,8 @@ def post(self, request, *args, **kwargs):
                     'auth_token': token.key,
                     'radius_user_token': radius_token.key,
                 }
+                token.user.last_login = timezone.now()
+                token.user.save(update_fields=['last_login'])
```
Comment (nemesisdesign) [inline openwisp_radius/api/views.py:509]: this code is repeated twice, please add a helper, eg:
```python
update_user_last_login(token.user)
```

File: openwisp_radius/api/views.py (hunk 1 above)
Comment (nemesisdesign) [reply openwisp_radius/api/views.py:473]: from now on I expect you to do this on every test you write, that's how it should be done :blush:

**Pull request merged**